src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -790,6 +791,13 @@ public void Save (System.IO.TextWriter stream)
 			var s = new StreamReader (ms).ReadToEnd ();
 			if (ApplicationName != null)
 				s = s.Replace ("${applicationId}", ApplicationName);
+			if (Placeholders != null)
+				foreach (var entry in Placeholders.Select (e => e.Split ('='))) {
```

Comment:
This should use [`string.Split(char[], int, StringSplitOptions)`](https://msdn.microsoft.com/en-us/library/ms131450(v=vs.110).aspx):

    foreach (var entry in Placeholders.Select (e => e.Split (new[]{'='}, 2, StringSplitOptions.None)) {
        if (entry.Length < 2) {
            log.LogWarning ($"Invalid application placeholders 
$(AndroidApplicationPlaceholders) value. Use `key1=value1;key2=value2` format. The specified value was: {string.Join (';', Placeholders})");
            continue;
        }
        s = s.Replace ("${" + entry [0] + "}", entry [1]);
    }

This overload removes the need to `string.Join ("=", entry.Skip(1)).